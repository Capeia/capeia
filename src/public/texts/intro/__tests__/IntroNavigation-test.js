// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<IntroNavigation>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('public/texts/intro/IntroNavigation')
    const Component = require('public/texts/intro/IntroNavigation').default

    const instance = renderer.create(<Component page={1} />)
    let tree = instance.toJSON()
    expect(tree).toMatchSnapshot()

    instance.update(<Component page={2} />)
    tree = instance.toJSON()
    expect(tree).toMatchSnapshot()

    instance.update(<Component page={3} />)
    tree = instance.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
