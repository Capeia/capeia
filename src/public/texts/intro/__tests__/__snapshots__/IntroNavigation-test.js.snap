// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroNavigation> renders correctly 1`] = `
<div>
  <h2
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    I
     | 
    <a
      href="/welcome/2"
    >
      II
    </a>
     | 
    <a
      href="/welcome/3"
    >
      III
    </a>
  </h2>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    />
    <div
      className="col-sm-6"
      style={
        Object {
          "textAlign": "right",
        }
      }
    >
      <a
        href="/welcome/2"
      >
        What Capeia is not ›
      </a>
    </div>
  </div>
</div>
`;

exports[`<IntroNavigation> renders correctly 2`] = `
<div>
  <h2
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <a
      href="/welcome"
    >
      I
    </a>
     | 
    II
     | 
    <a
      href="/welcome/3"
    >
      III
    </a>
  </h2>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    >
      <a
        href="/welcome"
      >
        ‹ Welcome to Capeia
      </a>
    </div>
    <div
      className="col-sm-6"
      style={
        Object {
          "textAlign": "right",
        }
      }
    >
      <a
        href="/welcome/3"
      >
        Our Motivation ›
      </a>
    </div>
  </div>
</div>
`;

exports[`<IntroNavigation> renders correctly 3`] = `
<div>
  <h2
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <a
      href="/welcome"
    >
      I
    </a>
     | 
    <a
      href="/welcome/2"
    >
      II
    </a>
     | 
    III
  </h2>
  <div
    className="row"
  >
    <div
      className="col-sm-6"
    >
      <a
        href="/welcome/2"
      >
        ‹ What Capeia is not
      </a>
    </div>
    <div
      className="col-sm-6"
      style={
        Object {
          "textAlign": "right",
        }
      }
    />
  </div>
</div>
`;
